x-pack/plugin/transform/src/main/java/org/elasticsearch/xpack/transform/persistence/TransformConfigManager.java

```diff
@@ -73,7 +73,7 @@
  *
  * Versioned Index:
  *
- * We wrap several indexes under 1 pattern: ".data-frame-internal-1", ".data-frame-internal-2", ".data-frame-internal-n" while
+ * We wrap several indexes under 1 pattern: ".data-frame-internal-1", ".data-frame-internal-2", ".transform-internal-n" while
```

Comment:
Should other mentions of "data-frame" in this comment be removed?

Reply:
it's still true as we provide BWC, however I can make this more clear that these are the old indexes